traits/has_traits.py

```diff
@@ -1964,7 +1947,7 @@ def _trait_view ( cls, name, view_element, default_name, view_elements,
                 result = view_elements.find( name )
                 if (result is None) and (handler is not None):
                     method = getattr( handler, name, None )
-                    if callable( method ):
+                    if six.callable( method ):
```

Comment:
I'll revert these changes; we don't need to support Python 3.0 or 3.1 (indeed, we don't support anything older than Python 3.5 at this point).